tests/lean/classAbbrev.lean.expected.out

```diff
@@ -0,0 +1,3 @@
+classAbbrev.lean:4:13-4:22: error: type expected
```

Comment:
When a test is supposed to fail, please add a comment on the `.lean` explaining why.